code/game/objects/effects/portals.dm

```diff
@@ -1,35 +1,102 @@
 /obj/effect/portal
 	name = "portal"
-	desc = "Looks unstable. Best to test it with the clown."
+	desc = "There's a hole in the sky... through which I can fly."

```

Comment:
You're free to submit a better replacement. But keep in mind that portals are now stable and see-through. I await your suggestions.